src/prompts/overlay.scss

```diff
@@ -1,11 +1,3 @@
-body.kumulos-overlay-blur {
```

Comment:
This makes the SDK have breaking CSS changes by renaming?

Agree semantics are better background mask, just not sure if anyone's overriding targeting these class names.

We could always just break it and assume anyone overriding will update / complain.

Thoughts?

Reply:
Ahh didn't consider anybody would be targeting this - could possibly add this back in and apply it to the div container redundantly from the overlay comp so its just 'there' for hooking into (albeit the css will be removed etc?) 🤔

Reply:
hm yeah that would work? like the component declares the old set as well... Hm, not sure

Reply:
OK so with this, looks like we're having both sets of classes to allow a non-breaking change.

- Do we need to keep duplicates of CSS rules?
- If we don't, do we need to add a `:not()` rule to the new ones to target older names? (maybe not)

Reply:
(A conversation around this outside this comment occurred resulting in the changes):

- BackgroundMask was used differently across the prompts and Overlay, however removed styling that was not present in the mask which was no longer needed against the Overlay directly, restructured scss slightly and enforced zindex's via vars and ensured stacking between overlay and prompts were as expected.
- Off the back of this but semi related edge case was also enhanced around the hiding of active prompts other than the one the user interacted with when the prompt was accepted.